packages/hint-babel-config/src/is-valid.ts

```diff
@@ -29,15 +29,14 @@ export default class BabelConfigIsValidHint implements IHint {
         };
 
         const invalidSchema = async (fetchEnd: BabelConfigInvalidSchema) => {
-            const { errors, prettifiedErrors, resource } = fetchEnd;
+            const { prettifiedErrors, resource } = fetchEnd;
 
             debug(`parse::error::babel-config::schema received`);
 
-            for (let i = 0; i < errors.length; i++) {
+            for (let i = 0; i < prettifiedErrors.length; i++) {
                 const message = prettifiedErrors[i];
-                const location = errors[i].location;
 
-                await context.report(resource, message, { location });
```

Comment:
Any particular reason `location` was removed? Without it we're not including where the error occurred...

Reply:
Yes, there is a reason. `errors` is the raw output from `ajv`, and with this PR `prettifiedErrors` can contain more than one `error`. So there is no way to know what location if the one for a `prettifiedError`.

Reply:
We may need a new type of object to return a list of then. Perhaps something to replace `prettifiedErrors` that includes both a `string` and a `location` derived from whichever was the first `error` contained in the entry?

All `error`s contained in a single entry in `prettifiedErrors` should originate from the same location anyway, correct?